rules/cross-platform/execution_potential_widespread_malware_infection.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2024/05/08"
+maturity = "production"
+updated_date = "2024/05/08"
+min_stack_comments = "ES|QL rule type is still in technical preview as of 8.13, however this rule was tested successfully"
+min_stack_version = "8.13.0"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule uses alert data to determine when a malware signature is triggered in multiple hosts. Analysts can use this
+to prioritize triage and response, as this can potentially indicate a widespread malware infection.
+"""
+from = "now-9m"
+language = "esql"
+license = "Elastic License v2"
+name = "Potential Widespread Malware Infection"
+references = [
+    "https://github.com/elastic/protections-artifacts/tree/main/yara/rules"
+]
+risk_score = 73
+rule_id = "28371aa1-14ed-46cf-ab5b-2fc7d1942278"
+severity = "high"
+tags = [
+    "Domain: Endpoint",
+    "Data Source: Elastic Defend",
+    "Use Case: Threat Detection",
+    "Tactic: Execution",
+    "Rule Type: Higher-Order Rule"
```

Comment:
is it?

Reply:
I think so, it uses alerts as the data source